tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -280,7 +282,24 @@ def restart_thermal_control_daemon(dut):
     config_reload(dut)
     assert 0, 'Wait thermal control daemon restart failed'
 
-
+def start_thermal_control_daemon(dut):
+    daemon_status, daemon_pid = dut.get_pmon_daemon_status(daemon_name)
+    if daemon_status != "RUNNING":
+        dut.start_pmon_daemon(daemon_name)
+        time.sleep(10)
```

Comment:
putting a 10 sec sleep is fine, but there is another API available that can help save a few secs 

eg: daemon/test_syseepromd.py:    wait_until(10, 2, 0, check_expected_daemon_status, duthost, expected_running_status) 

With this the max wait is 10sec, but check every 2 sec.